code/mod_table/mod_table.cpp

```diff
@@ -58,6 +58,11 @@ std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_p2;
 std::tuple<ubyte, ubyte, ubyte> Arc_color_emp_s1;
 bool Use_engine_wash_intensity;
 bool Ai_before_physics;
+SCP_vector<int> Required_render_ext;
```

Comment:
Why not use the enum type here? Same a few lines below. Since `CAPABILITY_BPTC` is already valid, that type should also be available.

Reply:
MSVC couldn't seem to find `gr_capability` despite the fact that Intellisense could detect it and didn't throw any errors, hence why I changed it to an int.

Reply:
What was the error in this case?

Reply:
I don't remember precisely but it resulted in an undefined symbol error towards the end of compilation.

Reply:
```
>C:\Users\tyler\Documents\GitHub\fs2open.github.com\code\mod_table/mod_table.h(52,19): error C2065: 'gr_capability': undeclared identifier (compiling source file C:\Users\tyler\Documents\GitHub\fs2open.github.com\code\scripting\api\objs\weaponclass.cpp)
>C:\Users\tyler\Documents\GitHub\fs2open.github.com\code\mod_table/mod_table.h(52,8): error C2923: 'SCP_vector': 'gr_capability' is not a valid template type argument for parameter 'T' (compiling source file C:\Users\tyler\Documents\GitHub\fs2open.github.com\code\scripting\api\objs\weaponclass.cpp)
>C:\Users\tyler\Documents\GitHub\fs2open.github.com\code\mod_table/mod_table.h(52,34): error C2062: type 'unknown-type' unexpected (compiling source file C:\Users\tyler\Documents\GitHub\fs2open.github.com\code\scripting\api\objs\weaponclass.cpp)
```